src/EPR.Payment.Service/Validations/RegistrationFees/Producer/ProducerRegistrationFeesRequestDtoValidator.cs

```diff
@@ -29,6 +29,10 @@ public ProducerRegistrationFeesRequestDtoValidator()
 
             RuleFor(x => x.ApplicationReferenceNumber)
                 .NotEmpty().WithMessage(ValidationMessages.ApplicationReferenceNumberRequired);
+
```

Comment:
What if the submission date is in future? Will this be validated?

Reply:
added validation for future dates